prow/jobs/cli/cli-k3d.yaml

```
@@ -0,0 +1,58 @@
+postsubmits:
+  kyma-project/cli:
+    - name: post-main-unstable-cli
+      annotations:
+        description: "Bump reconciler version used by CLI and publish the unstable CLI binaries"
+        owner: "jellyfish"
+        pipeline.clusterprovisioning: "reconciler"
+        pipeline.installer: "reconciler"
+        pipeline.test: "fast-integration"
+        pipeline.trigger: "pr-merge"
+        testgrid-create-test-group: "false"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "post-main-unstable-cli"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-kind-volume-mounts: "true"
+        preset-dind-enabled: "true"
+      skip_report: false
+      decorate: true
+      cluster: trusted-workload
+      max_concurrency: 10
+      branches:
+        - ^master$
+        - ^main$
+      extra_refs:
+        - org: kyma-project
+          repo: kyma
+          path_alias: github.com/kyma-project/kyma
+          base_ref: main
+        - org: kyma-incubator
+          repo: reconciler
+          path_alias: github.com/kyma-incubator/reconciler
+          base_ref: main
+      reporter_config:
+        slack:
+          channel: jellyfish-notifications
+      spec:
+        containers:
+          - image: europe-docker.pkg.dev/kyma-project/prod/testimages/e2e-dind-k3d:v20230612-dadc01a7
+            command: [ "/init.sh" ]
+            securityContext:
+              privileged: true
+              seccompProfile:
+                type: Unconfined
+              allowPrivilegeEscalation: true
+            args:
+              - "bash"
+              - "-c"
+              - |
+                chmod +x ./prow/publish-unstable-cli.sh
+                ./prow/publish-unstable-cli.sh
```

Chmod your script locally and push it to target repo. Then you don't need to chmod it in here.
```suggestion
              - ./prow/publish-unstable-cli.sh
```